src/shred_collector/service.zig

```diff
@@ -38,7 +38,7 @@ pub const ShredCollectorConfig = struct {
 /// Resources that are required for the Shred Collector to operate.
 pub const ShredCollectorDependencies = struct {
     allocator: Allocator,
-    logger: Logger,
+    logger: ScopedLogger(@typeName(@This())),
```

Comment:
This should not be a scoped logger

Reply:
[ffec062](https://github.com/Syndica/sig/pull/345/commits/ffec062b6033b04605c35af9e0aae033fa1752ff)